doc/COMPILING/COMPILING-XCODE.md

```diff
@@ -0,0 +1,53 @@
+# Disclaimer
+
+**WARNING**: Xcode build is **NOT** official and should be used for *iOS builds only*. This can only be done on OS X.
```

Comment:
```suggestion
**WARNING**: Xcode build should be used for *iOS builds only*. This can only be done on OS X.
```
What does not official mean here? This is a doc on the github of the project, how can it not be official?

Reply:
I guess it means it's not the primarily supported/maintained method of compiling the project.